packages/dev-server/src/adapter/cloudflare.ts

```diff
@@ -41,12 +43,16 @@ export const cloudflareAdapter: (options?: CloudflareAdapterOptions) => Promise<
     env: proxy.env,
     executionContext: proxy.ctx,
     onServerClose: async () => {
-      try {
-        await proxy.dispose()
-      } catch {
-        /**
-         * It throws an error if server is not running.
-         */
+      if (proxy !== undefined) {
+        try {
+          await proxy.dispose()
+        } catch (error) {
+          /**
+           * It throws an error if server is not running.
+           */
+        } finally {
+          proxy = undefined as Awaited<ReturnType<typeof getPlatformProxy<Env>>> | undefined
```

Comment:
You don't have to write the `as`. It will be:

```ts
proxy = undefined
```

Reply:
Thank you for checking! I have made the corrections.